ArduPlane/navigation.cpp

```diff
@@ -257,7 +257,7 @@ void Plane::calc_airspeed_errors()
 
     // when using the special GUIDED mode features for slew control, don't allow airspeed nudging as it doesn't play nicely.
 #if OFFBOARD_GUIDED == ENABLED
-    if (control_mode == &mode_guided && !is_zero(guided_state.target_airspeed_cm) && (airspeed_nudge_cm != 0)) {
+    if (control_mode == &mode_guided && is_positive(guided_state.target_airspeed_cm) && (airspeed_nudge_cm != 0)) {
```

Comment:
As it states from THROTTLE_NUDGE parameter description throttle input should affect target airspeed in auto-throttle modes and GUIDED mode is one of them. Also in Tuning Cruise Configuration it is written that this option should allow pilot to tweak target airspeed in AUTO and GUIDED modes. So I think the current behavior is not in line with texts and we should bring it in line.
Also on low processing power devises it will work from the box because this check will be skipped by `#if OFFBOARD_GUIDED == ENABLED`
But I have a concern about the reason of not passing plane sitl tests while passing quadplane tests.

Reply:
As I expected, failing climb test was because of not neutral throttle stick, making it neutral solved the issue.
As for fence issues, they also related to throttle stick position, and making throttle neutral solves them, but things are more complicated there...
In FenceRetRally test, RTL_RADIUS is set however WP_LOITER_RAD is used in autopilot in the first part of a test and by default it is 80. When autopilot try to circle 80 it can't, but the error is exactly 20 and testing for 100 passes :)
I tried setting WP_LOITER_RAD = 100 combined with NAVL1_LIM_BANK = 60, but still there is an error of 10 meters dew to lack of integral part in L1 controller while loitering.